sklearn/tree/tree.py

```diff
@@ -52,7 +52,7 @@
 CRITERIA_REG = {"mse": _criterion.MSE, "friedman_mse": _criterion.FriedmanMSE}
 
 DENSE_SPLITTERS = {"best": _splitter.BestSplitter,
-                   "presort-best": _splitter.PresortBestSplitter,
+                   "presort-best": _splitter.BestSplitter,
```

Comment:
You should defined instead `"presort-best": partial(_splitter.BestSplitter, presort=True)`. Though I am not sure it would work with your API.